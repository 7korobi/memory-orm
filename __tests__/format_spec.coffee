require "./models/index"
Mem = require "../src/index"

keys = [
  "tags", "faces", "chr_sets", "chr_npcs", "chr_jobs", 
  "locales", "randoms",
  "folders", "roles", "traps", "ables",
  "sow_roletables"]

keys.map (key)->
  describe key, ->
    test 'format snapshot', ->
      expect Mem.Query[key]._finder.format
      .toMatchSnapshot()
    undefined
