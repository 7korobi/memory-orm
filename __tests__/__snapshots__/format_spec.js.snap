// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ables format snapshot 1`] = `undefined`;

exports[`chr_jobs format snapshot 1`] = `undefined`;

exports[`chr_npcs format snapshot 1`] = `undefined`;

exports[`chr_sets format snapshot 1`] = `undefined`;

exports[`faces format snapshot 1`] = `undefined`;

exports[`folders format snapshot 1`] = `undefined`;

exports[`locales format snapshot 1`] = `undefined`;

exports[`randoms format snapshot 1`] = `undefined`;

exports[`roles format snapshot 1`] = `undefined`;

exports[`sow_roletables format snapshot 1`] = `undefined`;

exports[`tags format snapshot 1`] = `undefined`;
